charts/redpanda/templates/rbac.yaml

```diff
@@ -57,16 +79,29 @@ rules:
       - endpoints
       - events
       - limitranges
-      - persistentvolumeclaims
       - pods
       - pods/log
       - replicationcontrollers
       - resourcequotas
       - serviceaccounts
       - services
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - persistentvolumes
+      - persistentvolumeclaims
```

Comment:
I think i had at some point this set to clusterrole, but yeah you are right it is not needed here.

Reply:
Oh i see, if you check line 60, you will notice that it was set to cluster scope already, I moved it to line 97 because i needed to add patch, update, watch to this.

Reply:
I understand that side car need to have some new rules, but this particular place is used to define `rpk-bundle` ClusterRole rules. That's why I would like to propose to separate rpk-bundle specific rules that could be applied in helm chart after enabling some flag. 

My proposal would be to create new `Role` when the `statefulset.controllers.enabled` is set to `true`. What do you think about my proposal?

Reply:
Makes sense. I was trying to keep things a bit more tidy to avoid having to create more template files. sure thing.